cmd/drand-cli/control.go

```diff
@@ -15,6 +17,27 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
+func isWeak(password string) bool {
```

Comment:
Marginally better if the user is told what exactly is the problem with the password.

Reply:
+1